src/main/drivers/accgyro/accgyro.h

```diff
@@ -35,14 +35,12 @@
 #define MPU_I2C_INSTANCE I2C_DEVICE
 #endif
 
-#define GYRO_LPF_256HZ      0
-#define GYRO_LPF_188HZ      1
-#define GYRO_LPF_98HZ       2
-#define GYRO_LPF_42HZ       3
-#define GYRO_LPF_20HZ       4
-#define GYRO_LPF_10HZ       5
-#define GYRO_LPF_5HZ        6
-#define GYRO_LPF_NONE       7
+#define GYRO_HARDWARE_LPF_NORMAL       0
```

Comment:
perhaps a typedef enum here?

Reply:
Any real advantages either way?  I don't expect there will be any additions as they correlate to the hardware register settings.

Reply:
Fair enough. No real advantage except for not having to increment/define each one.